Resolve the issue of adb remount

tanminger opened the pull request:
If the secure boot is disabled, then ignore verify TOS failed.

tanminger committed:
Fix a bug about verify TOS partition failed when the device is unlocked.

If the secure boot is disabled, then ignore verify TOS failed.

Jira: None
Test: Test it in Joule, can boot to UI.

Signed-off-by: Ming Tan <ming.tan@intel.com>

android-ia-srv commented:
Commit message is invalid for commit id:  https://github.com/projectceladon/hardware-intel-kernelflinger/pull/1/commits/1098a3e20a4e2608b5bdc91117a1b0a7042dfc52 <br>Commit message should be as below: <br><br>Jira: Link to Jira [Default:None.]<br>Test: What tests are expected to pass <br>Signed-off-by: <Author/Contributor><br><br>For more details please refer to below wiki link: https://github.com/projectceladon/manifest/wiki/Contributions

tanminger committed:
Allow verify TOS error in unlocked state and in eng/userdebug build.

Some developer want to disable dm-verity.
Disable dm-verity need to unlock device and modify vbmeta. This will
cause verify vbmeta failed in the old code, and does not load Trusty.
This patch allow verify TOS error in unlocked state and in userdebug
build or eng build, then can load Trusty.

Jira: None.
Test: Test it in Joule, can boot to UI after unlock device,
      and can disable the dm-verity and adb remount.

Signed-off-by: Ming Tan <ming.tan@intel.com>
